src/core/components/documentParser/documentParser.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { DocumentParser } from './documentParser';
+
+/**
+ * DocumentParser component is used to render HTML or Markdown content in a read-only format.
+ */
+const meta: Meta<typeof DocumentParser> = {
+    title: 'Core/Components/DocumentParser',
+    component: DocumentParser,
+    tags: ['autodocs'],
+};
+
+type Story = StoryObj<typeof DocumentParser>;
+
+/**
+ * Default use case for the DocumentParser component with a basic string.
+ */
+export const Default: Story = {
+    args: {
+        stringDocument: 'Hello, world!',
```

Comment:
The stories needs to be updated to use the `document` property instead of `stringDocument`, right?